packages/types/src/factors.ts

```diff
@@ -53,6 +54,15 @@ export type BackupCodeFactor = {
   strategy: BackupCodeStrategy;
 };
 
+export type ResetPasswordCodeFactor = {
+  strategy: ResetPasswordCodeStrategy;
+  emailAddressId?: string;
+  safeIdentifier: string;
+  phoneNumberId?: string;
```

Comment:
❓ Can both `emailAddressId` and `phoneNumberId` be present? If not, can we make the type mutually exclusive?

Reply:
We decided that `phoneNumberId`  and `emailAddressId` will no longer be needed, and we are removing them. A [PR](https://github.com/clerkinc/javascript/pull/1079/files) is already open for this